templates/pages/login.jsx

```diff
@@ -53,21 +54,34 @@ var Login = React.createClass({
     // FIXME: totally not localized yet!
     var buttonText = "Log In";
     this.queryObj = Url.parse(window.location.href, true).query;
+
```

Comment:
My feelings here are that while it solve the 3+ optionally applied classname problem, we haven't quite hit that point, and I don't think it justifies adding a dependency just to avoid an if statement in our code.